Submitted By: Martin Schaffner <schaffner@gmx.li>
Date: 2004-04-02
Initial Package Version: 1.3.11
Upstream Status: Similar patch was rejected by author: http://lists.penguinppc.org/yaboot-devel/2004/yaboot-devel-200402/msg00001.html
Origin: Martin Schaffner
Description: Fixes cross-compilation.

--- a/Makefile	Wed Mar  3 02:19:32 2004
+++ b/Makefile	Wed Mar  3 02:23:25 2004
@@ -28,9 +28,15 @@
 #
 CROSS = 
 
+# compilation
+CC		:= $(CROSS)gcc
+LD		:= $(CROSS)ld
+AS		:= $(CROSS)as
+OBJCOPY		:= $(CROSS)objcopy
+
 # The flags for the yaboot binary.
 #
-YBCFLAGS = -Os $(CFLAGS) -nostdinc -Wall -isystem `gcc -print-file-name=include` -fsigned-char
+YBCFLAGS = -Os $(CFLAGS) -nostdinc -Wall -isystem `$(CC) -print-file-name=include` -fsigned-char
 YBCFLAGS += -DVERSION=\"${VERSION}\"	#"
 YBCFLAGS += -DTEXTADDR=$(TEXTADDR) -DDEBUG=$(DEBUG)
 YBCFLAGS += -DMALLOCADDR=$(MALLOCADDR) -DMALLOCSIZE=$(MALLOCSIZE)
@@ -92,12 +98,6 @@
 ifeq ($(CONFIG_FS_REISERFS),y)
 OBJS += second/fs_reiserfs.o
 endif
-
-# compilation
-CC		:= $(CROSS)gcc
-LD		:= $(CROSS)ld
-AS		:= $(CROSS)as
-OBJCOPY		:= $(CROSS)objcopy
 
 lgcc = `$(CC) -print-libgcc-file-name`
 
